src/features/movies/moviesSlice.js

```diff
@@ -37,4 +42,6 @@ export const selectMoviesList = (state) => state.movies.moviesList.payload;
 
 export const selectFetchStatus = (state) => state.movies.fetchStatus;
 
+export const selectMovieGenres = (state) => state.movies.moviesGenres.payload;
```

Comment:
Better variable name would be selectMoviesGenres - notice that it is what we are really selecting here:
`state.movies.--->>moviesGenres<<---.payload;`